src/index.js

```diff
@@ -1,18 +1,24 @@
 import './style.css';
+import { getScore, postScore } from './gameapi.js';
 
-const leaderboard = [
-  { name: 'Hannah', score: 98 },
-  { name: 'Kevin', score: 94 },
-  { name: 'Stuart', score: 81 },
-  { name: 'Matt', score: 87 },
-  { name: 'Alyssa', score: 90 },
-  { name: 'Jamie', score: 72 },
-];
+const refreshBtn = document.querySelector('.refresh-btn');
+const submitBtn = document.querySelector('.submit-btn');
+const userInput = document.querySelector('.name');
+const scoreInput = document.querySelector('.score');
 
-leaderboard.forEach((element) => {
-  const container = document.querySelector('#leaderboard');
-  const item = document.createElement('li');
-  item.classList.add('row');
-  item.textContent = `${element.name}:  ${element.score}`;
-  container.appendChild(item);
+submitBtn.addEventListener('click', async (e) => {
+  e.preventDefault();
+  const userInputF = userInput.value;
+  const scoreInputF = scoreInput.value;
+  await postScore(userInputF, scoreInputF);
+});
+
+function updateLeaderboard() {
```

Comment:
- [ ] Please avoid using the `function` keyword, instead use `arrow functions`, this is part of the module to make you a master in ES6 syntax 😄